lmdb-sys/src/bindings.rs

```diff
@@ -53,7 +53,7 @@ pub const MDB_BAD_VALSIZE: ::libc::c_int = -30781;
 pub const MDB_BAD_DBI: ::libc::c_int = -30780;
 pub const MDB_LAST_ERRCODE: ::libc::c_int = -30780;
 pub type mdb_mode_t = mode_t;
-pub type mdb_filehandle_t = ::libc::c_int;
+pub type mdb_filehandle_t = filehandle_t;
```

Comment:
This is auto-generated by bindgen. Are we sure this isn't something we should fix in bindgen?

Reply:
Actually, scratch my previous comment, we're blacklisting `mode_t` in our build script ourselves.

Please do the same here: https://github.com/mozilla/lmdb-rs/blob/master/lmdb-sys/bindgen.rs#L54